.travis.yml

```diff
@@ -8,6 +8,8 @@ before_install:
 - nvm install --lts
 script:
 - bundle exec rails db:{create,migrate} RAILS_ENV=test
+- bundle exec rails db:{create,migrate} RAILS_ENV=development
```

Comment:
Do we need this line of code here? I am not sure if it is actually working?

Reply:
good catch on that, just removed that line